shared/utility/math/filter/ParticleFilter.hpp

```diff
@@ -92,19 +94,13 @@ namespace utility::math::filter {
          * filter.
          * @param covariance The covariance of the multivariate normal distribution. Used for sampling the initial
          * particles in the filter.
-         * @param resample_method_ The method to use when resampling particles during the time update step.
-         * @param residual_method_ When using ResampleMethod::RESIDUAL for the resample method, this method will be used
-         * when the residual method needs to resample residual particles.
+         * @param resample_method_ The method to use when resampling particles during the time update step
```

Comment:
You are missing one

Reply:
I was avoiding changing the block comments in this PR, seeing as I am going to submit a PR updating all of them more thoroughly immediately after merging this one, but I'll fix these :shrug: